assets/js/main.js

```diff
@@ -28,26 +54,23 @@ const removeBook = (bookCard, id) => {
   bookCollection.removeBookFromCollection(id);
   updateStorage();
   bookCard.remove();
-  styleTable();
 };
 
 const addEventListenerToRemoveBtn = (button) => {
   button.addEventListener('click', (e) => removeBook(
-    e.target.parentNode,
-    parseInt(e.target.parentNode.dataset.bookid, 10),
+    e.currentTarget.parentNode,
+    parseInt(e.currentTarget.parentNode.dataset.bookid, 10),
```

Comment:
- [ ] **OPTIONAL**
- It's better to reset the form after submitting to allow a new entry. You can use the `.reset()`